modules/govuk/templates/node/s_graphite/remove_old_whisper_data.erb

```diff
@@ -8,7 +8,7 @@ find $whisper_dir -type f -mtime +730 -delete
 # Delete directories where none of the files inside have been modified in the
 # last 14 days. This effectively removes stats about machines which no longer
 # exist.
-for dir in $whisper_dir/*; do
+for dir in $whisper_dir/*/**; do
```

Comment:
Does this definitely work in the version of bash that's on these olde Ubuntu 14 machines? (Is `**` a bash 4 thing?)

Reply:
I've tried it by pasting into the command line on the integration graphite instance, and it works - that's how I found the 4911 subdirs mentioned above. But I'll deploy from this branch on integration and make sure it works in situ before merging :+1:

Reply:
Just ran this on integration:

```
alistairdavidson@ec2-integration-blue-graphite-ip-10-1-4-56:~$ find /opt/graphite/storage/whisper/ -type d -print | wc -l
47211
alistairdavidson@ec2-integration-blue-graphite-ip-10-1-4-56:~$ sudo /usr/local/bin/remove-old-whisper-data
alistairdavidson@ec2-integration-blue-graphite-ip-10-1-4-56:~$ find /opt/graphite/storage/whisper/ -type d -print | wc -l
22832
```

so that's actually more than half of the directories. The disk space used in that directory is _now_ :

```
$ du -sh /opt/graphite/storage/whisper/
246G	/opt/graphite/storage/whisper/
```

...so we've probably deleted about 250G of old data

Reply:
and that's also got rid of the old queues & exchanges from the RabbitMQ dashboard - only govuk_crawler_queue remains:
![image](https://user-images.githubusercontent.com/134501/220876823-92dc31e7-9720-4093-aa10-13dc65d47fda.png)